utils/schema/add-datasource-ids.ts

```diff
@@ -0,0 +1,84 @@
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+
+import type { DataSourceConfig } from '../types/DataSourceConfig';
+
+import { getAllDataSourceFiles } from '../lib/DataSource';
+
+const ARTIFACT_SCHEMA_PATH = './artifact.json';
+const CORE_DATASOURCE_IDS_PATH = './core-datasource-ids.json';
+const DATASOURCE_BASE_PATH = '../../';
```

Comment:
Not sure if its been a while in this repo, but i wasn't able to get this script to run right away.

I'm getting an error running the script using 

```bash
$ cd utils/
$ yarn run ts-node schema/add-datasource-ids.ts 
```

Error:
```bash
yarn run v1.22.19
$ /Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/node_modules/.bin/ts-node schema/add-datasource-ids.ts
[*] Found 0 community dataSources
Error: ENOENT: no such file or directory, open './core-datasource-ids.json'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at getCoreDataSourceIds (/Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/schema/add-datasource-ids.ts:38:8)
    at main (/Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/schema/add-datasource-ids.ts:69:19)
    at Object.<anonymous> (/Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/schema/add-datasource-ids.ts:83:3)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module.m._compile (/Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/jgregory/Developer/newrelic-external/newrelic-quickstarts/utils/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:981:32) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './core-datasource-ids.json'
}
```

I was able to run the script by updating these lines to utilize `path`

```ts
const ARTIFACT_SCHEMA_PATH = path.resolve(__dirname,'artifact.json');
const CORE_DATASOURCE_IDS_PATH = path.resolve(__dirname,'core-datasource-ids.json');
const DATASOURCE_BASE_PATH = path.resolve(__dirname, '..', '..');
```

Reply:
I believe the intention was to run `ts-node` from the same directory as the script (just verified, this still works). If we would prefer, I can update the script to work from the parent directory instead.

If it would be easier, I can also add this as a npm script.

Reply:
Ah i see, yeah totally works by just running `ts-node add-datasource-ids.ts` in the same location as the script. It just doesn't work with `yarn run ts-node` (which makes total sense), or in a parent directory. 

My original concern with not using `path` _was_ that we use ubuntu` to run our workflows. I thought it had a similar file system traversing as windows (which having `../` would break), but TIL its similar to mac. Also, doesn't seem like we're going to use this specific script in a workflow.

I could've verified at the time that the file structure is similar, so its not _really_ a concern anymore.

Having an npm script could be nice if we end up using this specific script in a workflow, but It doesn't seem like we are so we can skip that. Thanks for the clarification!

Reply:
I've added an npm script to make this a little easier.